### ISSUE TITLE
Example of folder structure

### ISSUE THREAD

**Issue by tkkarjal** [tkkarjal/magia#92]
An explicit example of the proper data structure (as well as the environmental variables) would be useful

**Comment by Ianyliu** [comment]
I agree. It would also be nice if BIDS was an accepted data structure.
